launchable/commands/record/build.py

```diff
@@ -63,9 +66,17 @@
     default=[],
     cls=KeyValueType,
 )
+@click.option(
+    '--link',
+    'links',
+    help="Set external link of title and url",
```

Comment:
[question]
Is this option requires both `title and url`, like dict format?

Reply:
Yes, like this ` --link build-url=https://example.launchable.com/job/1`

Reply:
ok